src/illuminant.rs

```diff
@@ -314,16 +314,16 @@ impl Illuminant {
     /// which takes a wavelength domain and spectral data as arguments.
```

Comment:
This documentation seems highly outdated? Talking about categories and stuff that the method does not actually deal with. I only fixed the issues in the code and left the documentation for now, since you probably knows better how to fix it.